src/color-picker/color-picker.scss

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+@import '../button/borderless';
+
+$iui-color-field-size: $iui-baseline * 19;
+
+
+@mixin iui-color-picker {
+  @include iui-dropdown;
+  max-width: ($iui-l * 10) + ($iui-xxs * 10) + ($iui-s * 9) + ($iui-sm * 2);
+  // max-width: (10 swatches per row) + (border) + (gaps) + (container padding)
+  max-height: ($iui-l * 10) + ($iui-xxs * 10) + ($iui-s * 9) + ($iui-baseline * 2);
+  // max-height: (10 rows) + (border) + (gaps) + (container padding)
+  box-sizing: border-box;
+  border-radius: $iui-border-radius;
+  display: flex;
+  padding: $iui-baseline $iui-sm;
+  user-select: none;
+  overflow-y: overlay;
+
+  &.iui-advanced {
+    max-width: ($iui-l * 7) + ($iui-xxs * 7) + ($iui-s * 6) + ($iui-sm * 2) + ($iui-s);
+    flex-wrap: wrap;
+  }
+}
+
+@mixin iui-color-swatch {
+  @include iui-focus;
```

Comment:
Seems like grey border is removed when color swatch was focused (focus, not focus-visible). It's really visible with white color on white background and grey color on darker background. 
![colorpickerfocus](https://user-images.githubusercontent.com/83585998/131314581-6722a903-e3e3-4cac-b78f-6463887a1de5.gif)

Reply:
Focusing white color has this background issue...
![image](https://user-images.githubusercontent.com/81580355/131316823-092f488d-f8f7-417a-9efb-f45eca277c0f.png)
![image](https://user-images.githubusercontent.com/81580355/131316850-36e65c0d-0b3a-4ebe-9264-fc2fc85cada9.png)

Wonder, if we could do something about it, but should not be a showstopper for this PR, I guess..

-----------------

Also, there is no focus style on the active color. For example, same situation on date picker:
active:
![image](https://user-images.githubusercontent.com/81580355/131317310-62f264b4-837c-438a-ab95-e64e42ddcedd.png)
focused active:
![image](https://user-images.githubusercontent.com/81580355/131317292-6ef7833e-8681-4e4a-a59a-ab078185de4d.png)

Reply:
The background issues were due to the inset box shadow getting removed when focusing so I changed the focus so that the inset box shadow remains when focused. 

When focusing on an active color the active box shadow (dark blue) gets replaced by the focus box shadow (light blue), the same thing happens when hovering over an active color (blue shadow gets replaced by the grey). I'm not sure what to do about this or if I should just leave it as is?

Reply:
> When focusing on an active color the active box shadow (dark blue) gets replaced by the focus box shadow (light blue), the same thing happens when hovering over an active color (blue shadow gets replaced by the grey).

For hovering, I don't think it should replace the blue shadow. That's why I said "active swatch could either use a double stripe or a different hover effect".(https://github.com/iTwin/iTwinUI/pull/253#discussion_r697716908)

The same could apply to focus too.

Reply:
Also you might want to add some basic JS in your HTML file to toggle the `iui-active` class on click.

Reply:
I added both a dark blue and light blue shadow when focusing on an active color.

Also you can now click on colors to toggle iui-active